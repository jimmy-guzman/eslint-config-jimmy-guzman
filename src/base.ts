export = {
  env: {
    browser: true,
    es2021: true,
    node: true,
  },
  parserOptions: {
    ecmaFeatures: {
      jsx: true,
    },
    jsxPragma: null,
    ecmaVersion: 12,
    sourceType: 'module',
  },
  overrides: [
    {
      files: ['.eslintrc.{js,cjs}'],
      env: {
        node: true,
      },
      parserOptions: {
        sourceType: 'script',
      },
      rules: {
        strict: 'off',
      },
    },
  ],
  ignorePatterns: [
    'node_modules/',
    'dist/',
    '.next/',
    'coverage/',
    'vite.config.ts.*.mjs',
    'storybook-static/',
  ],
}
