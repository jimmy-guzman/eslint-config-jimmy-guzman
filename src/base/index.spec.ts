import base from './index'

describe('base', () => {
  it('should export base rules config', () => {
    expect(base).toMatchSnapshot()
  })
})
