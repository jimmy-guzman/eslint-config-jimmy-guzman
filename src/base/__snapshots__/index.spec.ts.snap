// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`base should export base rules config 1`] = `
Object {
  "env": Object {
    "browser": true,
    "es2021": true,
    "node": true,
  },
  "parserOptions": Object {
    "ecmaFeatures": Object {
      "jsx": true,
    },
    "ecmaVersion": 12,
    "sourceType": "module",
  },
}
`;
